From 63e5a6343cd70f098b8679d0c9a587c3f5573de1 Mon Sep 17 00:00:00 2001
From: ValeraSh <9144287@gmail.com>
Date: Mon, 9 Dec 2013 23:41:22 +0300
Subject: [PATCH 22/56] Update russian translation to V17,

I removed all unnecessary rows from cm_strings.xml
Add translate activity indicators on status bar.
---
 res/values-ru/cr_strings.xml | 338 ++++++++++++++++---------------------------
 1 file changed, 125 insertions(+), 213 deletions(-)

diff --git a/res/values-ru/cr_strings.xml b/res/values-ru/cr_strings.xml
index 0761f58..3f41c86 100644
--- a/res/values-ru/cr_strings.xml
+++ b/res/values-ru/cr_strings.xml
@@ -279,21 +279,21 @@
     <string name="battery_style_reset_message">Сбросить все цвета индикатора батареи?</string>
    
     <!-- Animation Controls -->  
-    <string name="title_animation_controls">Пользовательская анимация системы</string>  
-    <string name="activity_open_title">Activity Open Animation</string>  
-    <string name="activity_close_title">Activity Close Animation</string>  
-    <string name="task_open_title">Task Open Animation</string>  
-    <string name="task_close_title">Task Close Animation</string>  
-    <string name="task_move_to_front_title">Move To Front Animation</string>  
-    <string name="task_move_to_back_title">Move To Back Animation</string>  
-    <string name="wallpaper_open_title">Wallpaper Open Animation</string>  
-    <string name="wallpaper_close_title">Wallpaper Close Animation</string>  
-    <string name="wallpaper_intra_open_title">Wallpaper Intra Open Animation</string>  
-    <string name="wallpaper_intra_close_title">Wallpaper Intra Close Animation</string>  
-    <string name="animation_duration_title">Animation Duration</string>  
-    <string name="title_animation_no_override">Prevent App Overrides</string>  
-    <string name="summary_animation_no_override">Prevent apps from overriding transitions.</string>  
-    <string name="animation_duration_default">Default</string>
+    <string name="title_animation_controls">Изменение системных анимаций</string>  
+    <string name="activity_open_title">Анимация открытия раздела</string>  
+    <string name="activity_close_title">Анимация закрытия раздела</string>  
+    <string name="task_open_title">Анимация запуска из раздела</string>  
+    <string name="task_close_title">Анимация возврата в раздел</string>  
+    <string name="task_move_to_front_title">Анимация выхода на передний план</string>  
+    <string name="task_move_to_back_title">Анимация возврата на задний план</string>  
+    <string name="wallpaper_open_title">Анимация возврата на домашний экран</string>  
+    <string name="wallpaper_close_title">Анимация запуска с домашнего экрана</string>  
+    <string name="wallpaper_intra_open_title">Анимация открытия последних приложений</string>  
+    <string name="wallpaper_intra_close_title">Анимация закрытия последних приложений</string>  
+    <string name="animation_duration_title">Длительность анимации</string>  
+    <string name="title_animation_no_override">Предотвратить изменение приложениями</string>  
+    <string name="summary_animation_no_override">Предотвратить изменение анимации приложениями</string>  
+    <string name="animation_duration_default">По умолчанию</string>
     <string name="header_summary_animation">@null</string>
 
     <!-- Lockscreen battery ring -->
@@ -421,102 +421,117 @@
     <string name="lockscreen_shorcuts_longpress_summary">Открывать пользовательское приложение при долгом нажатии на ярлык</string>
 
     <!-- Lock screen targets and shortcuts -->
-    <string name="picker_activities">Activities</string>
-    <string name="select_custom_activity_title">Select custom activity</string>
-    <string name="icon_picker_alarm">Alarm</string>
-    <string name="icon_picker_browser">Browser</string>
-    <string name="icon_picker_calendar">Calendar</string>
-    <string name="icon_picker_camera">Camera</string>
-    <string name="icon_picker_email">Email</string>
-    <string name="icon_picker_email2">Email Holo</string>
+    <string name="picker_activities">Действия</string>
+    <string name="select_custom_activity_title">Выберите своё действие</string>
+    <string name="icon_picker_alarm">Будильник</string>
+    <string name="icon_picker_browser">Браузер</string>
+    <string name="icon_picker_calendar">Календарь</string>
+    <string name="icon_picker_camera">Камера</string>
+    <string name="icon_picker_email">Почта</string>
+    <string name="icon_picker_email2">Почта Holo</string>
     <string name="icon_picker_facebook">Facebook</string>
-    <string name="icon_picker_gallery">Gallery</string>
-    <string name="icon_picker_google_small">Google (small)</string>
+    <string name="icon_picker_gallery">Галерея</string>
+    <string name="icon_picker_google_small">Google (малый)</string>
     <string name="icon_picker_googleplus">Google Plus</string>
     <string name="icon_picker_googletalk">Hangouts</string>
-    <string name="icon_picker_maps">Maps</string>
-    <string name="icon_picker_sms">Messaging</string>
-    <string name="icon_picker_movie">Movie</string>
-    <string name="icon_picker_music">Music</string>
-    <string name="icon_picker_nav_normal">Navigation</string>
-    <string name="icon_picker_phone">Phone</string>
-    <string name="icon_picker_podcast">Podcast</string>
+    <string name="icon_picker_maps">Карты</string>
+    <string name="icon_picker_sms">Сообщение</string>
+    <string name="icon_picker_movie">Видео</string>
+    <string name="icon_picker_music">Музыка</string>
+    <string name="icon_picker_nav_normal">Навигация</string>
+    <string name="icon_picker_phone">Телефон</string>
+    <string name="icon_picker_podcast">Подкасты</string>
     <string name="icon_picker_rss">Rss</string>
-    <string name="icon_picker_search">Search</string>
+    <string name="icon_picker_search">Поиск</string>
     <string name="icon_picker_twitter">Twitter</string>
     
     <!-- Halo -->
-    <string name="halo_settings_title">Halo</string>
-    <string name="halo_behavior_title">Halo Behavior</string>
-    <string name="halo_hide_title">Hide Halo after activity</string>
-    <string name="halo_hide_summary">Halo will appear for incoming tasks and scroll out of sight after a while</string>
-    <string name="halo_reversed_title">Reverse Halo</string>
-    <string name="halo_reversed_summary">Applies for task switching and moving gestures</string>
-    <string name="halo_state_title">Halo policy</string>
-    <string name="halo_state_summary">Enables black/white listing notifications</string>
-    <string name="halo_state_black">Black list</string>
-    <string name="halo_state_white">White list</string>
-    <string name="halo_pause_title">Pause active app</string>
-    <string name="halo_pause_summary">When opening an app with Halo the underlying app will get paused</string>
-    <string name="halo_ninja_title">Ninja mode</string>
-    <string name="halo_ninja_summary">Halo will disappear completely when empty</string>
-    <string name="halo_notification_msgbox_title">Enable message pop-up</string>
-    <string name="halo_notification_msgbox_summary">Show message box for incoming notifications</string>
-    <string name="halo_unlock_ping_title">Ping on unlock</string>
-    <string name="halo_unlock_ping_summary">Halo will ping when device is unlocked if new notifications were received</string>
-
-    <string name="halo_size_title">Halo size</string>
-    <string name="halo_size_summary">Defines how big Halo is going to be</string>
-    <string name="halo_size_miniscule">Miniscule</string>
-    <string name="halo_size_small">Small</string>
-    <string name="halo_size_default">Normal (default)</string>
-    <string name="halo_size_large">Large</string>
-    <string name="halo_size_gigantic">Gigantic</string>
-    <string name="halo_size_monster">Monster</string>
-
-    <string name="halo_notification_count_title">Notification count</string>
-    <string name="halo_notification_count_summary">Display notification counts on Halo</string>
-    <string name="halo_notification_count_none">None</string>
-    <string name="halo_notification_count_total">Total</string>
-    <string name="halo_notification_count_individual">Per app</string>
-    <string name="halo_notification_count_both">Both</string>
-
-    <string name="halo_msgbox_animation_title">Message box animation</string>
-    <string name="halo_msgbox_animation_summary">Halo message box animation for incoming notifications</string>
-    <string name="halo_msgbox_animation_none">None</string>
-    <string name="halo_msgbox_animation_flip">Flip</string>
-
-    <string name="halo_enabled_title">Enable HALO</string>
-    <string name="halo_enabled_summary">Completely enable or disable HALO</string>
+    <string name="halo_settings_title">Настройка Halo</string>
+    <string name="halo_behavior_title">Окружение HALO</string>
+    <string name="halo_hide_title">Скрыть HALO после активности</string>
+    <string name="halo_hide_summary">HALO будет появляться для входящих задач и исчезать спустя некоторое время</string>
+    <string name="halo_reversed_title">Отразить HALO</string>
+    <string name="halo_reversed_summary">Применяется для переключения задач и жестов</string>
+    <string name="halo_state_title">Политика HALO</string>
+    <string name="halo_state_summary">Включает чёрный и белый списки уведомлений</string>
+    <string name="halo_state_black">Чёрный список</string>
+    <string name="halo_state_white">Белый список</string>
+    <string name="halo_pause_title">Пауза активного приложения</string>
+    <string name="halo_pause_summary">При открытии приложения в HALO основное приложение будет приостановлено</string>
+    <string name="halo_ninja_title">Режим "Ниндзя"</string>
+    <string name="halo_ninja_summary">HALO будет исчезать полностью при отсутствии уведомлений</string>
+    <string name="halo_notification_msgbox_title">Включить всплывающие сообщения</string>
+    <string name="halo_notification_msgbox_summary">Показывать окно сообщения для входящих уведомлений</string>
+    <string name="halo_unlock_ping_title">Появление при разблокировке</string>
+    <string name="halo_unlock_ping_summary">HALO будет появляться при разблокировке, если были получены новые уведомления</string>
+
+    <string name="halo_size_title">Размер HALO</string>
+    <string name="halo_size_summary">Определяет насколько большим должен быть HALO</string>
+    <string name="halo_size_miniscule">Крошечный</string>
+    <string name="halo_size_small">Маленький</string>
+    <string name="halo_size_default">Обычный (по умолчанию)</string>
+    <string name="halo_size_large">Большой</string>
+    <string name="halo_size_gigantic">Гигантский</string>
+    <string name="halo_size_monster">Монстр</string>
+
+    <string name="halo_notification_count_title">Счётчик уведомлений</string>
+    <string name="halo_notification_count_summary">Отображает количество уведомлений в HALO</string>
+    <string name="halo_notification_count_none">Нет</string>
+    <string name="halo_notification_count_total">Всего</string>
+    <string name="halo_notification_count_individual">По приложениям</string>
+    <string name="halo_notification_count_both">Оба</string>
+
+    <string name="halo_msgbox_animation_title">Анимация окна сообщения</string>
+    <string name="halo_msgbox_animation_summary">Анимация окна сообщения HALO для входящих уведомлений</string>
+    <string name="halo_msgbox_animation_none">Нет</string>
+    <string name="halo_msgbox_animation_flip">Переворот</string>
+
+    <string name="halo_enabled_title">Включение HALO</string>
+    <string name="halo_enabled_summary">Полностью включает или отключает HALO</string>
 
     <!-- Label for are-halo-notifications-allowed checkbox in app details [CHAR LIMIT=20] -->
-    <string name="app_halo_label_black">Blacklist from HALO</string>
-    <string name="app_halo_label_white">Whitelist from HALO</string>
+    <string name="app_halo_label_black">Чёрный список HALO</string>
+    <string name="app_halo_label_white">Белый список HALO</string>
 
     <!-- ListView animation -->
     <string name="bars_and_menus_category_system_title">Система</string>
     <string name="listview_animation_title">Выбрать анимацию для списков</string>
-    <string name="listview_off">Выкл.</string>
+    <string name="listview_off">Выключена</string>
     <string name="listview_wave_left">Волна (Слева)</string>
     <string name="listview_wave_right">Волна (Справа)</string>
-    <string name="listview_alpha">Альфа</string>
-    <string name="listview_scale">Масштаб</string>
-    <string name="listview_stack_top">Стек (Вверху)</string>
-    <string name="listview_stack_bottom">Стек (Внизу)</string>
-    <string name="listview_unfold">Раскрывается</string>
-    <string name="listview_fold">Сворачивается</string>
-    <string name="listview_translate_left">Перевод (Слева)</string>
-    <string name="listview_translate_right">Перевод (Справа)</string>
+    <string name="listview_alpha">Прозрачность</string>
+    <string name="listview_scale">Увеличение</string>
+    <string name="listview_stack_top">Стэк (Сверху)</string>
+    <string name="listview_stack_bottom">Стэк (Снизу)</string>
+    <string name="listview_unfold">Раскладывание</string>
+    <string name="listview_fold">Складывание</string>
+    <string name="listview_translate_left">Перемещение (Слева)</string>
+    <string name="listview_translate_right">Перемещение (Справа)</string>
 
     <!-- ListView interpolator -->
     <string name="listview_interpolator_title">Выбрать интерполятор для списков</string>
-    <string name="listview_accelerate_interpolator">accelerate_interpolator</string>
-    <string name="listview_decelerate_interpolator">decelerate_interpolator</string>
-    <string name="listview_accelerate_decelerate_interpolator">accelerate_decelerate_interpolator</string>
-    <string name="listview_anticipate_interpolator">anticipate_interpolator</string>
-    <string name="listview_overshoot_interpolator">overshoot_interpolator</string>
-    <string name="listview_anticipate_overshoot_interpolator">anticipate_overshoot_interpolator</string>
-    <string name="listview_bounce_interpolator">bounce_interpolator</string>
+    <string name="listview_accelerate_interpolator">Ускоренный</string>
+    <string name="listview_decelerate_interpolator">Замедленный</string>
+    <string name="listview_accelerate_decelerate_interpolator">Ускоренный-замедленный</string>
+    <string name="listview_anticipate_interpolator">Предсказывающий</string>
+    <string name="listview_overshoot_interpolator">Ошибающийся</string>
+    <string name="listview_anticipate_overshoot_interpolator">Предсказывающий-ошибающийся</string>
+    <string name="listview_bounce_interpolator">Отскакивающий</string>
+
+    <!-- Lockscreen style settings -->
+    <string name="lockscreen_style_reset_message">Сбросить все цвета экрана блокировки?</string>
+    <string name="lockscreen_icon_title">Иконка замка в слайдере</string>
+    <string name="lockscreen_lock_icon_custom">Пользовательская иконка</string>
+    <string name="lockscreen_lock_icon_default">Иконка по умолчанию</string>
+    <string name="lockscreen_colorize_handle_title">Изменить цвет замка</string>
+    <string name="lockscreen_colorize_handle_summary">Пользовательская картинка будет определять цвет блокировки</string>
+    <string name="lockscreen_style_title">Стиль блокировки</string>
+    <string name="lockscreen_frame_color_title">Цвет рамки</string>
+    <string name="lockscreen_frame_color_summary">Цвет рамки и фона экрана разблокировки</string>
+    <string name="lockscreen_lock_color_title">Цвет замка</string>
+    <string name="lockscreen_lock_color_summary">Цвет иконки замка в слайдере разблокировки</string>
+    <string name="lockscreen_dots_color_title">Цвет точек</string>
+    <string name="lockscreen_dots_color_summary">Цвет светящихся точек вокруг слайдера разблокировки</string>
 
     <!-- Lockscreen widget settings -->
     <string name="lockscreen_widgets_title">Виджеты</string>
@@ -538,12 +553,12 @@
     <string name="ad_text_title">Текст уведомлений</string>
     <string name="ad_text_summary">Показывать текст уведомления вокруг слайдера.</string>
     <string name="ad_all_notifications_title">Все уведомления</string>
-    <string name="ad_all_notifications_summary">Показывать все уведомленя, включая текущие и не прекращающиеся.</string>
+    <string name="ad_all_notifications_summary">Показывать все уведомления, в том числе постоянные, не отфильтрованные уведомления.</string>
     <string name="ad_hide_low_priority_title">Скрыть низкий приоритет</string>
     <string name="ad_hide_low_priority_summary">Скрыть уведомления с низким приоритетом, например от Google Now</string>
     <string name="ad_pocket_mode_title">"Карманный" режим</string>
-    <string name="ad_pocket_mode_summary">Показывать активные уведомления, когда вы достаете устройство из кармана.</string>
-    <string name="ad_redisplay_title">Обновить уведомления</string>
+    <string name="ad_pocket_mode_summary">Показывать уведомления активного дисплея, когда устройство извлекается из кармана.</string>
+    <string name="ad_redisplay_title">Показывать уведомления повторно</string>
     <string name="ad_sunlight_mode_title">Режим "Солнечный свет"</string>
     <string name="ad_sunlight_mode_summary">Инвертировать цвета, если вы находитесь в ярко освещённом месте, например под прямым солнечным светом.</string>
     <string name="ad_customizations_title">Настройки</string>
@@ -667,12 +682,15 @@
     <string name="pref_network_mode_title">Режимы сети</string>
     <string name="pref_screentimeout_mode_title">Время до отключения экрана</string>
     <string name="pref_ring_mode_title">Режимы звука</string>
-    <string name="screentimeout_mode1">15с | 1м | 5м | Никогда</string>
-    <string name="screentimeout_mode2">30с | 2м | 5м | Никогда</string>
+    <string name="pref_music_mode_title">Режим музыки</string>
+    <string name="screentimeout_mode1">15сек | 1мин | 5мин | Никогда</string>
+    <string name="screentimeout_mode2">30сек | 2мин | 5мин | Никогда</string>
     <string name="sound_mode_silent">Беззвучный</string>
     <string name="sound_mode_vibrate">Вибрация</string>
     <string name="sound_mode_sound">Звук</string>
     <string name="sound_mode_soundVibrate">Звук и вибросигнал</string>
+    <string name="music_tile_mode_background">Показывать обложку альбома</string>
+    <string name="music_tile_mode_tracktitle">Показывать название трека</string>
 
     <!-- QuickSettings: Quick Pulldown -->
     <string name="title_quick_pulldown">Быстрый доступ к плиткам</string>
@@ -680,119 +698,13 @@
     <string name="quick_pulldown_off">Выкл.</string>
     <string name="quick_pulldown_left">Левый</string>
     <string name="quick_pulldown_right">Правый</string>
-	
-    <!-- Display : Rotation -->
-    <string name="display_rotation_title">Поворот экрана</string>
-    <string name="display_rotation_disabled">Отключен</string>
-    <string name="display_rotation_unit">градусов</string>
-    <string name="display_rotation_category_title">Режимы вращения</string>
-    <string name="display_rotation_0_title">0 градусов</string>
-    <string name="display_rotation_90_title">90 градусов</string>
-    <string name="display_rotation_180_title">180 градусов</string>
-    <string name="display_rotation_270_title">270 градусов</string>
-	
-    <!-- INTERFACE Settings -->
-    <!-- Settings main menu category heading. Interface (Launcher, Themes, System). [CHAR LIMIT=40] -->
-    <string name="header_category_interface">Персонализация</string>
-	
-    <!-- Themes settings -->
-    <string name="themes_settings_title">Темы</string>
-	
-    <!--- Lock clock -->
-    <string name="lock_clock_title">Виджет «cLock»</string>
-    <string name="lock_clock_summary">Просмотр или изменение настроек отображения виджета «cLock» на главном экране или экране блокировки</string>
-	
-    <!-- Lock screen widgets -->
-    <string name="lockscreen_widgets_category_title">Виджеты</string>
-    <string name="lockscreen_enable_widgets_title">Виджеты приложений</string>
-	
-    <!-- Lights settings screen, notification light settings -->
-    <string name="display_lights_settings_title">Экран и индикация</string>
-    <string name="lights_category_title">Индикация</string>
-    <string name="notification_pulse_title">Индикатор событий</string>
-    <string name="notification_light_general_title">Основные</string>
-    <string name="notification_light_applist_title">Приложения</string>
-    <string name="notification_light_phonelist_title">Телефон</string>
-    <string name="notification_light_use_custom">Собственные значения</string>
-    <string name="notification_light_enabled">Включено</string>
-    <string name="notification_light_disabled">Выключено</string>
-    <string name="notification_light_default_value">По умолчанию</string>
-    <string name="notification_light_missed_call_title">Пропущенный вызов</string>
-    <string name="notification_light_voicemail_title">Голосовая почта</string>
-	
-    <!-- Backlight brightness settings screen -->
-    <string name="adaptive_backlight_title">Адаптивная подсветка</string>
-    <string name="adaptive_backlight_summary">Интеллектуальное изменение яркости подсветки экрана для сохранения заряда аккумулятора</string>
-	
-    <!-- Battery light settings -->
-    <string name="battery_light_title">Индикация батареи</string>
-    <string name="battery_low_pulse_title">Индикация разрядки батареи</string>
-    <string name="battery_light_list_title">Цвета</string>
-    <string name="battery_light_low_color_title">Батарея разряжена</string>
-    <string name="battery_light_medium_color_title">Заряжается</string>
-    <string name="battery_light_full_color_title">Полностью заряжена</string>
-	
-    <!-- Hardware keys -->
-    <string name="button_settings">Кнопки</string>
-    <string name="hardware_keys_enable_custom_title">Разрешить пользовательские кнопки</string>
-    <string name="hardware_keys_home_key_title">Кнопка «Домой»</string>
-    <string name="hardware_keys_menu_key_title">Кнопка «Меню»</string>
-    <string name="hardware_keys_volume_keys_title">Кнопки громкости</string>
-    <string name="hardware_keys_short_press_title">Действие на короткое нажатие</string>
-    <string name="hardware_keys_long_press_title">Действие на долгое нажатие</string>
-    <string name="hardware_keys_double_tap_title">Действие на двойное нажатие</string>
-    <string name="hardware_keys_show_overflow_title">Показывать кнопку меню</string>
-    <string name="hardware_keys_show_overflow_summary">Показывать кнопку меню (вертикальное троеточие) в приложениях</string>
-    <string name="hardware_keys_show_overflow_toast_enable">Кнопка меню может не отображаться до перезапуска приложений</string>
-    <string name="hardware_keys_show_overflow_toast_disable">Кнопка меню может отображаться до перезапуска приложений</string>
-    <string name="hardware_keys_action_nothing">Нет действия</string>
-    <string name="hardware_keys_action_menu">Меню приложения</string>
-    <string name="hardware_keys_action_app_switch">Запущенные приложения</string>
-    <string name="hardware_keys_action_search">Поисковый помощник</string>
-    <string name="hardware_keys_action_voice_search">Голосовой поиск</string>
-    <string name="hardware_keys_action_in_app_search">Поиск в приложении</string>
-    <string name="home_wake_title">Пробуждение кнопкой «Домой»</string>
-    <string name="home_wake_summary">Нажатие кнопки «Домой» пробуждает устройство</string>
-    <string name="volume_wake_title">Пробуждение кнопками громкости</string>
-    <string name="volume_wake_summary">Нажатие на кнопки громкости пробуждает устройство</string>
-    <string name="volbtn_music_controls_title">Управление проигрывателем</string>
-    <string name="volbtn_music_controls_summary">Когда экран выключен, долгое нажатие кнопок громкости переключает музыкальные треки</string>
-    <string name="swap_volume_buttons_title">Поворот кнопок громкости</string>
-    <string name="swap_volume_buttons_summary">Менять назначение кнопок регулировки громкости согласно ориентации экрана</string>
-	
-    <!-- Hardware tunables - Key backlight -->
-    <string name="button_backlight_title">Подсветка</string>
-    <string name="button_backlight_enabled">Подсветка кнопок</string>
-    <string name="keyboard_backlight_enabled">Подсветка клавиатуры</string>
-    <string name="button_backlight_seekbar_title">Яркость кнопок</string>
-    <string name="keyboard_backlight_seekbar_title">Яркость клавиатуры</string>
-    <string name="backlight_timeout_title">Длительность подсветки</string>
-    <string name="backlight_timeout_unlimited">Не выключать</string>
-    <plurals name="backlight_timeout_time">
-        <item quantity="one"><xliff:g id="seconds">%d</xliff:g> секунду</item>
-        <item quantity="other"><xliff:g id="seconds">%d</xliff:g> секунд</item>
-    </plurals>
-    <string name="backlight_summary_disabled">Выключена</string>
-    <string name="backlight_summary_enabled_with_timeout">Включена <xliff:g id="timeout">%s</xliff:g></string>
-    <string name="backlight_summary_enabled">Включена всегда</string>
-	
-    <!--  Other device settings -->
-    <string name="more_device_controls_title">Ещё...</string>
-    <string name="more_device_settings_title">Другие настройки устройства</string>
-    <string name="sensors_motors_title">Датчики и вибрация</string>
-	
-    <!-- Hardware tunables - Vibrator intensity -->
-    <string name="vibrator_intensity_title">Мощность вибросигнала</string>
-    <string name="vibrator_warning">Установка значений от %1$d и выше не рекомендуется</string>
-	
-    <!-- Category title for headset specific Settings. [CHAR LIMIT=40] -->
-    <string name="headset_category_title">Наушники</string>
-	
-    <!-- Launch music player when headset is connected -->
-    <string name="headset_connect_player_title">Запуск проигрывателя</string>
-    <string name="headset_connect_player_summary">Запускать проигрыватель при подключении наушников</string>
-	
-    <!-- Performance settings -->
-    <string name="performance_settings_title">Производительность</string>
-	
+
+    <!-- security settings menu unlock -->
+    <string name="unlock_menu_unlock_title">Разблокировка кнопкой "Меню"</string>
+    <string name="unlock_menu_unlock_summary">Нажатие на кнопку "Меню" разблокирует устройство.</string>
+
+    <!-- Status Bar Activity Indicators -->
+    <string name="show_activity_indicators_on_status_bar_title">Индикатор сетевой активности</string>
+    <string name="show_activity_indicators_on_status_bar_summary">Показывать индикатор сетевой активности в строке сосояния</string>
+
 </resources>
-- 
1.8.4.msysgit.0

